### PR TITLE
svelte store improvements

### PR DESCRIPTION
**Pull request by TSKsmiley**
co authored by:
<NybyDK@users.noreply.github.com>

### CODE REVIEW

File: src/lib/classes/NetworkManager.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+		if (node) {
+			this._nodes = this._nodes.filter((n) => n.id !== nodeId);
+			this.nodeMap.delete(nodeId);
+		}
+
+		this.fastUpdate();
+	}
+
+	addConnection(input: { source: Node; target: Node }) {
+		// Ensure that the two nodes are not the same
```
Comment (BaBrixx) [inline src/lib/classes/NetworkManager.ts:51]: Delete this

**Review by BaBrixx — CHANGES_REQUESTED** [review]
This is shit

File: src/lib/classes/NetworkManager.ts (hunk 1 above)
Comment (TSKsmiley) [reply src/lib/classes/NetworkManager.ts:51]: yeah i agree the code is self documenting

Comment (TSKsmiley) [reply src/lib/classes/NetworkManager.ts]: image

**Review by Stormlighter — APPROVED** [review]

**Pull request merged**